vpr/src/pack/cluster_router.h

```diff
@@ -23,7 +23,9 @@ bool try_intra_lb_route(t_lb_router_data* router_data, int verbosity, t_mode_sel
 void reset_intra_lb_route(t_lb_router_data* router_data);
 
 /* Accessor Functions */
```

Comment:
Make a proper Doxygen comment.  "Accessor functions" seems out of date -- suggest deleting that line.